app/controllers/ops_controller/diagnostics.rb

```diff
@@ -644,19 +676,19 @@ def diagnostics_set_form_vars
       @selected_server ||= MiqServer.find(@sb[:selected_server_id]) # Reread the server record
       if @sb[:selected_server_id] == my_server.id
         if @sb[:active_tab] == "diagnostics_evm_log"
-          @log = Vmdb::Loggers.contents($log)
+          @log = fetch_local_log("evm")
```

Comment:
TODO `Vmdb::Loggers.contents(log)` will only fetch the last 1000 lines

Reply:
That's intentional for the UI - it can be massive otherwise.

Reply:
Yup just needed the reminder that I had to implement this on the journald side, done

Reply:
Done